ecs-agent/netlib/model/ecscni/eni_config.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ecscni
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-agent/ecs-agent/netlib/model/networkinterface"
+)
+
+// ENIConfig contains all the information needed to invoke the eni plugin
+type ENIConfig struct {
```

Comment:
The new version of the ENI model is renamed to NetworkInterface. Shall we make the corresponding change here as well?

Reply:
Will update!

Reply:
Right I think that makes sense. I am OK keeping it as is.